content/ecosystem/servers/servers.toml

```
@@ -5,7 +5,7 @@ author = "Matrix.org team"
 maturity = "Stable"
 language = "Python"
 licence = "Apache-2.0"
```

Please update the license as well :)

```suggestion
licence = "AGPL-3.0-or-later"
```